### ISSUE TITLE
Support page numbering in LaTeX template

### ISSUE THREAD

**Issue by twsh** [jgm/pandoc#10380]
**Describe your proposed improvement and the problem it solves.**

Some other templates have a 'pagenumbering' variable which changes how page numbers appear. The LaTeX template could easily have this too:

```
$if(pagenumbering)$
\pagenumbering{$pagenumbering$}
$endif$
```

I was thinking that this could be in common.latex where the page styles are done.

The manual should then list the options LaTeX allows (arabic, alph, Alph, roman, Roman).

I can make a PR, unless this is a bad idea for some reason.

One issue is that this won't work with the Book, Memoir, and Scrbook classes. The `\frontmatter` and `\mainmatter` commands override the setting of page numbering.

**Describe alternatives you've considered.**

**Comment by tarleb** [comment]
I tend to think that this is a bit too niche for the default template.

**Comment by jgm** [comment]
Agreed, it's an uncommon thing to want, and people can insert raw tex when needed using `--include-in-header`.
